mautic/mautic
# [UX] Quick filters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation-new/pull/208 
| Issue(s) addressed                     | Fixes #... 



## Description

Dependency:
- #13689 

Users often face challenges when trying to quickly filter and locate specific entries in the Users and Forms sections, which can result in inefficiencies. 

This PR addresses the issue by adding clickable quick filters for "Has full system access," "Active,", "Inactive" and "Has result" statuses, based on the existing search commands. 

These filters enhance the user experience by allowing users to swiftly narrow down their search results, saving time and improving navigation efficiency, especially for those managing large datasets (with dozens of users etc).

Users:
![image](https://github.com/mautic/mautic/assets/116097999/7879fd47-2f7a-4542-917b-9c0c90372bbf)

Forms:
![image](https://github.com/mautic/mautic/assets/116097999/b55eb82f-7951-431e-a254-7a32ed41cdf0)







---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Users > Create 1 with full system access, another Active and another not Active
3. Go back to the Users list and click on each filter
4. Go to Forms > Create one 
5. Go back to the list, click on the With responses filter and see your form disappear because it has no submissions
